content/api/webhooks.apib

```diff
@@ -4,54 +4,81 @@ description: Manage webhooks, which enable you to receive push updates of the ra
-| auth_token        | string | Authentication token to present in the `X-MessageSystems-Webhook-Token` header of POST requests to target | no | Use this token in your target application to confirm that data is coming from the Webhooks API. <br />example: `5ebe2294ecd0e0f08eab7690d2a6ee69`<br /><br /><span class="label label-danger"><strong>Deprecated</strong></span> in favor of the `auth_type` field. |
 
-__**The SparkPost webhooks API uses MaxMind software [MaxMind License](https://www.maxmind.com/download/geoip/database/LICENSE.txt)**__
+### Authentication
+
+You can use authentication to protect your webhook consumer and verify that only SparkPost is sending you data. We support both OAuth 2 and Basic authentication. When using OAuth 2, you need to set `auth_request_details`. The `auth_credentials` will be set by automatically from this. If you use Basic authentication, you need to set `auth_credentials` with a valid username and password.
+
+## Webhooks Object
+
++ Data Structure: Attributes
+    + name (string) - User-friendly name for webhook
+    + target (string) - URL of the target to which to POST event batches. 
+    + events (array)  - Array of event types this webhook will receive. Use the [Events Documentation](#webhooks-get-events-documentation-get) endpoint to list the available event types.
+    + active (boolean) - The status of the webhook. When false, the target will no longer receive batches.
+    + custom_headers (object) - Object of custom headers to be used during POST requests to target.
+    + auth_type (enum) - Type of authentication to be used during POST requests to target.
+        + none
+        + basic
+        + oauth2
+    + auth_request_details (object) - Object containing details needed to request authorization credentials for OAuth 2.
+    + auth_credentials (object) - Object containing credentials needed to make authorized POST requests for Basic Authentication.
+    + auth_token (string)
+        <span class="label label-danger"><strong>Deprecated</strong></span> in favor of the `auth_type` field. <br /> Authentication token to present in the `X-MessageSystems-Webhook-Token` header of POST requests to target.
++ Sample
+        
+    ```
+    {
+        "name": "Example webhook",
+        "target": "http://client.example.com/example-webhook",
+        "events": ["delivery", "injection", "open", "click"],
+        "active": true,
+        "custom_headers": {
+            "x-api-key": "abcd"
+        },
+        "auth_type": "basic",
+        "auth_credentials": {
+            "username": "basicauthuser",
+            "password": "mypassword"
+        }
+    }
+    ```
 
-## Events Documentation [/webhooks/events/documentation]
+__**The SparkPost Webhooks API uses MaxMind software [MaxMind License](https://www.maxmind.com/download/geoip/database/LICENSE.txt)**__
```

Comment:
Did you want to `**`s to render? Also can we not move this to the end? I think we can